README.md

```diff
@@ -152,6 +152,26 @@ Do this if you want to use the remote server for building and deploying.
 
     Where the `$user` shell variable contains the name of your shell account on the remote machine.
 
+### Configuring `geth-<testnet/mainnet>-vm` machine
+This step configure geth mainnet or geth testnet instance on the remote server.
+
+The first playbook creates machine, ip and disk. The `$geth_instance` variable controls which ethereum blockchain instance will be created(`geth-testnet/geth-mainnet`).
```

Comment:
Please add information that the playbooks are meant to be run from `concent-builder` or another machine that is authorized to run `gcloud` commands on our GKE project.